classes/Domain/Services/TalkRating/TalkRatingContext.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace OpenCFP\Domain\Services\TalkRating;
+
+use OpenCFP\Domain\Model\TalkMeta;
+use OpenCFP\Domain\Services\Authentication;
+
+class TalkRatingContext
+{
+    public static function getTalkStrategy(string $strategy, Authentication $auth) : TalkRatingStrategy
+    {
+        $strategy = strtolower($strategy);
+        switch ($strategy) {
+            case 'yesno':
+                return new YesNoRating(new TalkMeta, $auth);
+            default:
+                return new YesNoRating(new TalkMeta, $auth);
+        }
+    }
```

Comment:
Given the default case is the same as 'yesno', can't this just be

return new YesNoRating(new TalkMeta, $auth);

Reply:
That would also do the trick, but i made it this way as a set-up for adding more rating options.